kujira/app/services/enevt-handler-service.js

```diff
@@ -0,0 +1,33 @@
+
+import Ember from 'ember';
+var run= Ember.run;
+var socket;
+var hostName ='';
```

Comment:
Value should be taken from config/environment.js: ENV.APP.API_HOST and used in io.connect directrly.